tests/gen/Generated.hack

```diff
@@ -150,6 +157,56 @@ final class User extends \Slack\GraphQL\Types\ObjectType {
   }
 }
 
+final class OutputTypeTest extends \Slack\GraphQL\Types\ObjectType {
```

Comment:
Weird that sort order seems non-deterministic for DefinitionFinder. We should sort our objects before generating code.

Reply:
Yea, I write out the top level objects first but would be good to sort this too. Add an issue!

Reply:
Yea, I write out the top level objects first but would be good to sort this too. Add an issue!